src/compiler/crystal/macros/methods.cr

```diff
@@ -595,6 +595,18 @@ module Crystal
         end
       when "identify"
         interpret_argless_method(method, args) { StringLiteral.new(@value.tr(":", "_")) }
+      when "includes?"
+        interpret_one_arg_method(method, args) do |arg|
+          case arg
+          when CharLiteral
+            piece = arg.value
+          when StringLiteral
```

Comment:
I took the code from the existing macro methods. `starts_with?`, `ends_with?` `+`, .. all do the same. Figured it was best to be consistent.